sdk/documenttranslator/ai-document-translator-rest/README.md

```diff
@@ -89,32 +89,12 @@ az cognitiveservices account keys list --name "resource-name" --resource-group "
 
 Once you have an API key and endpoint, you can use the `AzureKeyCredential` class to authenticate the client as follows:
 
-```javascript
-const DocumentTranslator = require("@azure-rest/ai-document-translator");
+```ts snippet:ReadmeSampleCreateClient_APIKey
+import DocumentTranslator from "@azure-rest/ai-document-translator";
 
 const client = DocumentTranslator("<endpoint>", { key: "<API key>" });
 ```
 
-#### Using an Azure Active Directory Credential
-
-Client API key authentication is used in most of the examples, but you can also authenticate with Azure Active Directory using the [Azure Identity library][azure_identity]. To use the [DefaultAzureCredential][defaultazurecredential] provider shown below,
-or other credential providers provided with the Azure SDK, please install the `@azure/identity` package:
-
-```bash
-npm install @azure/identity
-```
-
-You will also need to [register a new AAD application][register_aad_app] and grant access to Document Translator by assigning the `"Managed Application Contributor Role"` role to your service principal (note: other roles such as `"Owner"` will not grant the necessary permissions, only `"Managed Application Contributor Role"` will suffice to run the examples and the sample code).
-
-Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: `AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`.
-
-```javascript
-const DocumentTranslator = require("@azure-rest/ai-document-translator");
-const { DefaultAzureCredential } = require("@azure/identity");
-
-const client = DocumentTranslator("<endpoint>", new DefaultAzureCredential());
```

Comment:
should we still keep an example of using DefaultAzureCredential, or is Entra ID not supported?